src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1259,6 +1259,21 @@ private bool IsValidOverrideReturnType(Symbol overridingSymbol, TypeWithAnnotati
                 }
             }
 
+            TypeWithAnnotations getNotNullIfNotNullOutputType(TypeWithAnnotations outputType, ImmutableHashSet<string> notNullIfParameterNotNull)
+            {
+                for (var i = 0; i < overriddenParameters.Length; i++)
+                {
+                    var overridingParam = overridingParameters[i + overridingMethodOffset];
+                    var overriddenParam = overriddenParameters[i];
```

Comment:
>var overriddenParam [](start = 20, length = 19)

Consider inlining in call below to avoid accessing if not needed.

Reply:
Done